packages/app/scripts/build.js

```diff
@@ -130,17 +131,28 @@ function build(previousSizeMap) {
     const info = stats.toJson();
 
     if (stats.hasErrors()) {
-      console.error('Failed to create a production build. Reason:');
+      console.error('Error creating a production build:');
       console.error(info.errors);
+      ls - a;
       process.exit(1);
     }
 
     if (stats.hasWarnings()) {
-      console.warn('Warnings in Webpack build:');
-      console.warn(info.warnings);
+      console.warn(chalk.yellow('Build warnings:'));
+      // console.warn(info.warnings);
+      stats.compilation.warnings.forEach(({ name, message }) => {
+        console.warn(chalk.yellow(`${name}: ${message}\n`));
+      });
     }
 
-    console.log(chalk.green('Compiled successfully.'));
+    const took = Date.now() - start;
+
+    console.log(
+      chalk.green(
+        `Built ${stats.hasWarnings() ? 'with warnings ' : ''}in ${took /
+          1000}s.`
+      )
+    );
     console.log();
 
     // console.log('File sizes after gzip:');
```

Comment:
We can start showing this again after we remove `CopyWebpackPlugin`, I commented this because it was also showing all files copied by the plugin (+50,000 files), which made the build actually slow because of slow stdout.

Reply:
I think we can / should do something more involved as a separate step in the build process.

Reply:
But how does that work, anyway, as in CI we don't have the previous build's files?

Reply:
We do! We cache them between builds in CircleCI.